CHANGELOG.md

```diff
@@ -2,9 +2,19 @@
 
 Converts VCF variants into a [FHIR Genomics Diagnostic Report](http://hl7.org/fhir/uv/genomics-reporting/index.html). 
 
-## Additions
+## Added
 * In scope are simple variants (SNVs, Indels), along with zygosity and phase relationships, for autosomes, sex chromosomes, and mitochondrial DNA.
 * Supports VCF file (text-based or bgzipped) and optionally tabix files for query.
 * Supports genome build ('GRCh37' or 'GRCh38');
 * Optionally supports a query region in the form of .bed or dictionary that indicates the region(s) to convert.
 * Optionally supports inclusion of  'region-studied' observations that detail which portions of the conversion region were studied, and of those studied regions, which portions were deemed uncallable.
+
+# v0.0.17 (2021-06-07)
+
+## Added
+* Exposed a new parameter to configure ratio for determining Homoplasmic vs Heteroplasmic.
+
+## Fixed
+* Removed 'Human reference sequence assembly version' component from variant observation.
+* Excluded records with incorrect CHROM values in vcf file ([#19](https://github.com/elimuinformatics/vcf2fhir/issues/19)).
+* Case where conversion region is not studied.

```

Comment:
It's a good practice to add issue number at the end like we did for `Excluded records with incorrect CHROM value....` can we add it for other two as well.